cpp/tests/io/parquet_test.cpp

```diff
@@ -462,7 +464,59 @@ TYPED_TEST(ParquetWriterNumericTypeTest, SingleColumnWithNulls)
   CUDF_TEST_EXPECT_TABLES_EQUAL(expected, result.tbl->view());
 }
 
-TYPED_TEST(ParquetWriterChronoTypeTest, Chronos)
+TEST_F(ParquetWriterTest, Durations)
```

Comment:
In my last review I missed this case: We may need to check for duration types with nulls. Maybe you can add it inside this test by reusing the input columns: 
 * Write a file without nulls first, test reading it, then
 * Set random null masks for the columns, write another file and test reading it separately.